lnd.go

```diff
@@ -871,8 +871,12 @@ func waitForWalletPassword(grpcEndpoints, restEndpoints []string,
 	if registeredChains.PrimaryChain() == litecoinChain {
 		chainConfig = cfg.Litecoin
 	}
+	macaroonFiles := []string{
```

Comment:
Add a comment why we are passing the macaroon files to the unlocker service here.

Reply:
Fixed.